### ISSUE TITLE
Kernel GC breakage

### ISSUE THREAD

**Issue by erights** [Agoric/agoric-sdk#3562]
@FUDCo , after rebase-ing https://github.com/Agoric/agoric-sdk/pull/3525 to current master, I still get the following error:

```
  test/virtualObjects/test-representatives.js:469

   468:   }                                                                     
   469:   t.deepEqual(remainingVOs, {                                           
   470:     'v1.vs.vom.o+1/2': '{"label":{"body":"\\"thing #2\\"","slots":[]}}',

  Difference:

    {
      'v1.vs.vom.o+1/2': '{"label":{"body":"\\"thing #2\\"","slots":[]}}',
  +   'v1.vs.vom.o+1/2.refCount': '0 0',
      'v1.vs.vom.o+1/3': '{"label":{"body":"\\"thing #3\\"","slots":[]}}',
  -   'v1.vs.vom.o+1/4': '{"label":{"body":"\\"thing #4\\"","slots":[]}}',
  -   'v1.vs.vom.o+1/5': '{"label":{"body":"\\"thing #5\\"","slots":[]}}',
  -   'v1.vs.vom.o+1/6': '{"label":{"body":"\\"thing #6\\"","slots":[]}}',
  +   'v1.vs.vom.o+1/3.refCount': '1 0',
  +   'v1.vs.vom.o+1/8': '{"label":{"body":"\\"thing #8\\"","slots":[]}}',
  +   'v1.vs.vom.o+1/9': '{"label":{"body":"\\"thing #9\\"","slots":[]}}',
    }

  › test/virtualObjects/test-representatives.js:469:5
```

See https://github.com/Agoric/agoric-sdk/pull/3525/checks?check_run_id=3196077205
and https://github.com/Agoric/agoric-sdk/pull/3525/checks?check_run_id=3196077794

This PR used to work on a very recent master, so the kernel gc problem is new.

**Comment by erights** [comment]
Note that the PR is a one line change, so it is trivial to try it out on whatever instrumented version you have.

**Comment by FUDCo** [comment]
Closed by #3566
